Documentation/engineering/updating-ryujit.md

```diff
@@ -2,12 +2,12 @@
 
 Following steps are necessary to pick up a new version of RyuJIT code generation backend from CoreCLR:
 
-1. From the master branch of the CoreCLR repo, copy header files that are part of the contract with RyuJIT from `src\inc` on the CoreCLR side, to `src\JitInterface\src\ThunkGenerator` on the CoreRT side.
+1. From the master branch of the https://github.com/dotnet/runtime/ repo, copy header files that are part of the contract with RyuJIT from `src\coreclr\src\inc` on the CoreCLR side, to `src\JitInterface\src\ThunkGenerator` on the CoreRT side.
 2. Inspect the diffs
     1. If an enum was modified, port the change to the managed version of the enum manually.
     2. If a JitInterface method was added or changed, update `src\JitInterface\src\ThunkGenerator\ThunkInput.txt` and run the generation script next to the file to regenerate `CorInfoBase.cs` and `jitinterface.h`. Update the managed implementation of the method in `CorInfoImpl.cs` manually.
     3. If the JitInterface GUID was updated (`JITEEVersionIdentifier` in `corinfo.h`), update it in `src\Native\jitinterface\jitwrapper.cpp`
-3. Determine the latest Microsoft.NETCore.Jit package to use
+3. Determine the latest Microsoft.NETCore.Jit package to use (**TODO: This package is no longer built. Has to be retooled**)
```

Comment:
This can be updated to mention Microsoft.NETCore.App.Runtime package now that Michal updated it:

https://github.com/dotnet/corert/pull/8001/files#diff-d7bea50ef95c95c2ed222c77540b7f8aR15

Reply:
Do you happen to know what's the place to find the latest version of the packages? I stole a version number from a maestro update in the Extensions repo: https://github.com/dotnet/extensions/pull/3008, but not sure if that's what we would want to document.

Reply:
I run `dotnet restore` against a dummy .csproj like this:

```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>netcoreapp2.1</TargetFramework>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Microsoft.NETCore.App.Runtime.win-x64" Version="5.*" />
  </ItemGroup>
</Project>
```

It won't restore succesfuly, but the error message is going to have the latest version.

Maybe we can add this .csproj next to this doc and link it from here?

Reply:
Does this mean that instead of going for the build, latest version of package should be obtained from the Nuget ? And this project is a mean for getting that package.

Reply:
Yes, we would replace the instructions that start with "Go to https://dnceng.visualstudio.com/internal/_build?definitionId=244 and select the latest successful *scheduled build* of the master branch" and instead say we should just restore this project and look for the latest version in the error message.